tools/compute_font_metrics.py

```diff
@@ -77,109 +75,227 @@
 <!doctype html>
 <html>
   <head>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-beta1/jquery.min.js"></script> 
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.25.4/js/jquery.tablesorter.min.js"></script> 
     <style>
-      html, body{
-        font-family:Arial, sans-serif;
+      body {
+        font-family: sans-serif;
       }
-      div.filename, div.darkess{
+      div.filename, div.darkess {
         width:100px;
-        color:gray;
+        color: lightgrey;
         font-size:10px;
       }
-      img{
-        margin-left:150px;
+      img {
+        margin-left:1em;
+      }
+      thead {
+        background: grey;
+      }
+      td {
+        border-left: 1px lightgrey solid;
+        white-space: nowrap;
+      }
+      tr.old {
+        background: lightgrey;
+      }
+      tr.new {
+        background: white;
       }
     </style>
   </head>
   <body>
-    %s
+      <table id="myTable" class="tablesorter">
+        <thead>
+          <tr>
+              <td>Filename</td>
+              <td>GFN</td>
+              <td>Weight</td>
+              <td>Weight Int</td>
+              <td>Width</td>
+              <td>Width Int</td>
+              <td>Angle</td>
+              <td>Angle Int</td>
+              <td>Usage</td>
+              <td>Image Weight</td>
+              <td>Image Width</td>
+          </tr>
+        </thead>
+        %s
+      </table>
+    <script type="text/javascript">
+      $(document).ready(function() 
+          { 
+              $("#myTable").tablesorter(); 
+          } 
+      ); 
+    </script>
   </body>
 </html>
 """
 
 # When outputing the debug HTML, this is used to show a single font.
 ENTRY_TEMPLATE = """
-<div>
-  <div class='darkness'>%s</div>
-  <div class='filename'>%s</div>
-  <img src="data:image/png;base64,%s" />
-</div>
+<tr class="%s">
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+  <td>%s</td>
+</tr>
 """
 
+# Normalizes a set of values from 0 to target_max
+def normalize_values(properties, target_max=1.0):
+  max_value = 0.0
+  for i in range(len(properties)):
+    val = float(properties[i]['value'])
+    max_value = max(max_value, val)
+  for i in range(len(properties)):
+    properties[i]['value'] *= (target_max/max_value)
+
+# Maps a list into the integer range from target_min to target_max
+# Pass a list of floats, returns the list as ints
+# The 2 lists are zippable
+def map_to_int_range(weights, target_min=1, target_max=10):
```

Comment:
this would be a bit better refactoring:

```python
def map_to_int_range(weights, target_min=1, target_max=10):
  weights_as_int = []
  weights_ordered = sorted(weights)
  min_value = float(weights_ordered[0])
  max_value = float(weights_ordered[-1])
  target_range = (target_max - target_min)
  float_range = (max_value - min_value)
  for weight in weights:
    weight = target_min + int(target_range * ((weight - min_value) / float_range))
    weights_as_int.append(weight)
  return weights_as_int
```